clients/tabby-agent/src/AgentConfig.ts

```diff
@@ -70,6 +77,11 @@ export const defaultAgentConfig: AgentConfig = {
       experimentalStripAutoClosingCharacters: false,
       maxPrefixLines: 20,
       maxSuffixLines: 20,
+      experimentalDeclarations: {
+        enabled: false,
+        maxSnippets: 5,
+        maxChars: 2000,
```

Comment:
this shall be applied on each individual snippet (e.g 500), rather than total max chars